randomness cargo test error

substrate-developer-hub/recipes/pallets/randomness$ cargo test
    Updating git repository `https://github.com/encointer/substrate-fixed.git`
   Compiling sp-api v2.0.0
error[E0432]: unresolved import `sp_core::to_substrate_wasm_fn_return_value`
  --> ~/.cargo/registry/src/github.com-1ecc6299db9ec823/sp-api-2.0.0/src/lib.rs:54:9
   |
54 | pub use sp_core::to_substrate_wasm_fn_return_value;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `to_substrate_wasm_fn_return_value` in the root

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: could not compile `sp-api`

To learn more, run the command again with --verbose.

--- a/pallets/randomness/Cargo.toml
+++ b/pallets/randomness/Cargo.toml
@@ -29,7 +29,6 @@ sp-std = { version = '2.0.0', default-features = false }
 
 [dev-dependencies]
 pallet-randomness-collective-flip = { version = '2.0.0', default-features = false }
-pallet-timestamp = { version = '2.0.0', default-features = false }
 sp-io = { version = '2.0.0', default-features = false }
 
 [features]

--- a/pallets/randomness/src/tests.rs
+++ b/pallets/randomness/src/tests.rs
@@ -93,7 +93,9 @@ fn generate_works() {
 			System::events(),
 			vec![EventRecord {
 				phase: Phase::Initialization,
-				event: TestEvent::randomness(Event::RandomnessConsumed(H256::zero(), H256::zero(),)),
+				event: TestEvent::randomness(
+					Event::RandomnessConsumed(H256::zero(), H256::zero(),)
+				),
 				topics: vec![],
 			}]
 		);

